Corona-Warn-App/src/test/java/de/rki/coronawarnapp/covidcertificate/validation/core/DccValidationTest.kt

```diff
@@ -0,0 +1,102 @@
+package de.rki.coronawarnapp.covidcertificate.validation.core
+
+import de.rki.coronawarnapp.covidcertificate.validation.core.business.wrapper.createGeneralRule
+import de.rki.coronawarnapp.covidcertificate.validation.core.country.DccCountry
+import de.rki.coronawarnapp.covidcertificate.validation.core.rule.DccValidationRule
+import de.rki.coronawarnapp.covidcertificate.validation.core.rule.EvaluatedDccRule
+import io.kotest.matchers.shouldBe
+import org.joda.time.Instant
+import org.junit.jupiter.api.Test
+import testhelpers.BaseTest
+
+class DccValidationTest : BaseTest() {
+
+    private val userInput = ValidationUserInput(
+        DccCountry("PT"),
+        Instant.now()
```

Comment:
Please avoid `Instant.now()` (hardcode timestamps) as it could make tests flaky.